index.html

```diff
@@ -33,15 +33,19 @@ <h2 class="subtitle" id="hero-subtitle">
     </header>
 
     <main>
-        <section class="section">
-            <div class="field has-addons" id="search-bar">
-                <input type="text" class="input control" id="search-bar-input"
-                    placeholder="Search book and movie titles...">
-                <button type="submit" id="search-button">
-                    <img src="./assets/images/search-icon.png" alt="magnifying glass icon" id="search-icon">
-                </button>
+        <section class="inputContainer">
```

Comment:
this needs a class of .section, not .inputContainer.  .section is a bulma class that is styled automatically. I will merge though..please clean this up